### ISSUE TITLE
API

### ISSUE THREAD

**Issue by kent1313** [digitaleagle/moses#10]
We need an intermediate program that will communicate between the database and the UI. Right now, no such program exists. The program needs to be initiated. From there, we can work together on what all the program needs to relay from the database to the UI.

**Comment by digitaleagle** [comment]
I don't think we need an intermediate program ... Firebase should act as both.  I just committed the firebase code, so that's a start.
